core/pom.xml

```diff
@@ -80,7 +80,7 @@
     <dependency>
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
-      <version>2.7</version>
+      <version>2.5</version>
```

Comment:
Maybe add a comment about why we are doing this here? In case it takes a while for us to move to 2.7